docker/dropin_env_base_jdk_ubi/build_image.sh

```diff
@@ -26,7 +26,7 @@ IMAGE_ORG_DATAROBOTDEV=datarobotdev
 
 IMAGE_REPO=dropin-env-base-jdk
 
-IMAGE_TAG=ubi8.8-py3.11-jdk11.0.22-drum1.13.0-mlops9.2.8
+IMAGE_TAG=ubi8.8-py3.11-jdk11.0.25-drum1.13.0-mlops9.2.8
```

Comment:
This is a residue left from a prev PR. Fix it ti have code updated.

Doesn't really matter whether tag has `22` or `25` . Image already pushed so I'll use tag with `22`